pysilcam/__main__.py

```diff
@@ -605,7 +623,7 @@ def collect_rts(settings, rts, stats_all):
         rts.to_csv(filename)
 
 
-def writeCSV(datafilename, stats_all):
+def write_stats(datafilename, stats_all):
```

Comment:
I wonder if this function should be in process.py? I could be used externally from __main__ in some cases.

Reply:
Done